### ISSUE TITLE
rdmp-cli: blank lines printed to console when running dle file load

### ISSUE THREAD

**Issue by rkm** [HicServices/RDMP#195]
**Describe the bug**

When running e.g. `./rdmp dle --command run -l 12`, many blank lines are printed to the console. Potentially could be one line for each row loaded? Or per log statement, but for some reason nothing is shown.

**To Reproduce**

1. From the NSH system, run the above command when there is a file available for loading
1. Observe output

**Expected behavior**

Not this.

**Screenshots**

N/A

**RDMP Version**

rdmp-cli `v4.1.8`

**Error with Stack Trace**

N/A

**Database Engine**

MySQL

**Additional context**

N/A

**Comment by tznind** [comment]
Logging output is determined by `NLog.config` which sits in the binary directory (if you don't have this file it will log only to the RDMP logging database).  I have observed this newline issue before but not sure if it has been resolved since 4.1.8 or not.  I've looked at the code and I can't see any empty Console.WriteLine statements.  It's possible it is in FAnsi or something upstream.

The default NLog.config that ships currently will log to a `./logs` directory
